readthedocs/templates/projects/project_edit.html

```diff
@@ -14,9 +14,6 @@
 
 {% block project_edit_content %}
   <form method="post" action=".">{% csrf_token %}
-    <p class="empty">
-      {% blocktrans with advanced_url="../advanced" %}You can change how your project is built in your <a href="{{ advanced_url}}">Advanced Settings</a>.{% endblocktrans %}
-    </p>
```

Comment:
Shouldn't we link to the settings page instead of removing this block?

Reply:
This is the settings page

Reply:
I don't follow you. Can you expand on this?

Reply:
project-edit is the settings page. Having the link would link to the same page.